DuckDuckGo/UserText.swift

```diff
@@ -1412,4 +1412,20 @@ But if you *do* want a peek under the hood, you can find more information about
             public static let skip = NSLocalizedString("onboarding.highlights.fireDialog.cta.skip", value: "Skip", comment: "The title of the fire button CTA to skip erasing the data.")
         }
     }
+
+    public enum AddToDockOnboarding {
+        public enum Buttons {
+            static let addToDuckTutorial = NSLocalizedString("contextual.onboarding.addToDock.buttons.tutorial", value: "Show Me How", comment: "Button on the last screen of the onboarding, it will show the Add to Duck tutorial.")
```

Comment:
"Add to Dock tutorial”
I’d try to be a bit more descriptive:
Button at the end of the browser onboarding on click it shows the user how to add the application to the mac doc

or something like that

Reply:
I think I forgot finishing the description. good catch!